refactor: Update log schema

JordanClemons opened the pull request:
<!--
    PLEASE REVIEW BEFORE OPENING
    PR guidelines:
    - Delete this comment so the preview begins with your description
    - Make sure not to include any internal links
    - Please fill out all sections where applicable!
    - PR title should be in the format <prefix>: <short description>
        - for a reminder of what prefixes are available, see here: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines
    - Add your tester as a reviewer and let them know when changes are ready for them to start looking at
    - Add "N/A" under any sections that are not applicable
-->

## Description

Update how logging is formatted.

<!-- Describe your changes and what problem they solve -->

## Screenshots

<!-- Add any relevant screenshots of the change or fix -->

## Testing instructions

<!--
    Include any useful information that will help with testing this change specifically, if applicable
    General testing setup can be omitted - this should focus on setup unique to this PR
-->

JordanClemons committed:
Change endpoint

JordanClemons committed:
Change domain

JordanClemons committed:
Payoad translation

JordanClemons committed:
Fix snapshots

JordanClemons committed:
Update snapshots

JordanClemons committed:
Merge develop

JordanClemons committed:
Update getFeatures to support input features

paypal-rosman commented on src/library/zoid/modal/component.js:
```diff
@@ -192,10 +192,10 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
                             index,
                             refIndex,
                             et: 'CLICK',
-                            event_type: 'click',
```
just want to make sure this was a click before and now is modal_rendered?

paypal-rosman commented on src/library/zoid/modal/component.js:
```diff
@@ -164,9 +164,9 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
-                            event_type: 'click',
-                            link: linkName,
-                            src: src ?? linkName
+                            event_type: 'modal_rendered',
```
just want to make sure this was a click before and now is modal_rendered?

JordanClemons replied:
Correct

paypal-rosman commented:
Approved but have some comments

JordanClemons committed:
Add api to domain

JordanClemons committed:
Fix api domain

JordanClemons committed:
Fix unit test

surekhaw committed:
feat: update DE LT disclaimer (#1053)

surekhaw committed:
feat: update pm legal disclaimer and disclosure (#1054)

yeul committed:
prerender modal tweaks for modxo (#1055)

gillycheesesteak committed:
chore: cut release (#1056)

semantic-release-bot committed:
chore(release): 1.57.0 [skip ci]

## [1.57.0](https://github.com/paypal/paypal-messaging-components/compare/v1.56.0...v1.57.0) (2024-03-05)

### Features

* update DE LT disclaimer ([#1053](https://github.com/paypal/paypal-messaging-components/issues/1053)) ([7c34362](https://github.com/paypal/paypal-messaging-components/commit/7c343624ff194791b96fb0bf88791bc3978781a7))
* update pm legal disclaimer and disclosure ([#1054](https://github.com/paypal/paypal-messaging-components/issues/1054)) ([be9563f](https://github.com/paypal/paypal-messaging-components/commit/be9563feb5d57dfebf2be23275d2fdfe24d38b0e))

erikacolette29 committed:
feat: logic to handle merchant including placement and page-type parameters the SDK (#1050)

yeul committed:
chore: cut release (#1059)

chore: cut release

semantic-release-bot committed:
chore(release): 1.58.0 [skip ci]

## [1.58.0](https://github.com/paypal/paypal-messaging-components/compare/v1.57.0...v1.58.0) (2024-03-12)

### Features

* logic to handle merchant including placement and page-type parameters the SDK ([#1050](https://github.com/paypal/paypal-messaging-components/issues/1050)) ([f109fc7](https://github.com/paypal/paypal-messaging-components/commit/f109fc7db0db9215ab888100ff19480e3ffc7345))

surekhaw committed:
feat: fix preapproval pi4 xo modal (#1062)

grablack committed:
chore: cut release #1063

semantic-release-bot committed:
chore(release): 1.59.0 [skip ci]

## [1.59.0](https://github.com/paypal/paypal-messaging-components/compare/v1.58.0...v1.59.0) (2024-03-19)

### Features

* fix preapproval pi4 xo modal ([#1062](https://github.com/paypal/paypal-messaging-components/issues/1062)) ([2448735](https://github.com/paypal/paypal-messaging-components/commit/244873576fbf4535f042c40bc49a2c61bf8f590c))

Seavenly committed:
fix: setup global state from script in modal setup (#1058)

jadutter committed:
fix: modified DE header margin so it will not overlap with the svg (#1060)

yeul committed:
refactor: tweak close styles (#1064)

surekhaw committed:
feat: improve ql xo ax (#1065)

grablack committed:
chore: cut release #1067

semantic-release-bot committed:
chore(release): 1.60.0 [skip ci]

## [1.60.0](https://github.com/paypal/paypal-messaging-components/compare/v1.59.0...v1.60.0) (2024-03-26)

### Features

* improve ql xo ax ([#1065](https://github.com/paypal/paypal-messaging-components/issues/1065)) ([a6e3d4f](https://github.com/paypal/paypal-messaging-components/commit/a6e3d4fabd64333e62966fde92a56438593a3791))

### Bug Fixes

* modified DE header margin so it will not overlap with the svg ([#1060](https://github.com/paypal/paypal-messaging-components/issues/1060)) ([b6e5374](https://github.com/paypal/paypal-messaging-components/commit/b6e53740b5946d113911416267550177be7f013a))
* setup global state from script in modal setup ([#1058](https://github.com/paypal/paypal-messaging-components/issues/1058)) ([18ff80e](https://github.com/paypal/paypal-messaging-components/commit/18ff80eb5e00efdadc94d1baeb489410b0126f7d))

### Code Refactoring

* tweak close styles ([#1064](https://github.com/paypal/paypal-messaging-components/issues/1064)) ([4df4302](https://github.com/paypal/paypal-messaging-components/commit/4df430262dcfbe6b974572fbc432e7b19f321bf2))

yeul committed:
feat: add support for buttons messages (#1057)

merlinpaypal committed:
chore: cut release (#1069)

semantic-release-bot committed:
chore(release): 1.61.0 [skip ci]

## [1.61.0](https://github.com/paypal/paypal-messaging-components/compare/v1.60.0...v1.61.0) (2024-04-02)

### Features

* add support for buttons messages ([#1057](https://github.com/paypal/paypal-messaging-components/issues/1057)) ([b3700c9](https://github.com/paypal/paypal-messaging-components/commit/b3700c968afb486fef6809bc948a20194fc049d2))

yeul committed:
refactor: update QL checkout Pay Monthly and Pay in 4 modals (#1068)

yeul committed:
feat: support onError modal callback (#1070)

yeul committed:
chore: cut release (#1071)

semantic-release-bot committed:
chore(release): 1.62.0 [skip ci]

## [1.62.0](https://github.com/paypal/paypal-messaging-components/compare/v1.61.0...v1.62.0) (2024-04-09)

### Features

* support onError modal callback ([#1070](https://github.com/paypal/paypal-messaging-components/issues/1070)) ([8a0a2eb](https://github.com/paypal/paypal-messaging-components/commit/8a0a2eb88d275a8cc6cb5215c683b08e696940de))

### Code Refactoring

* update QL checkout Pay Monthly and Pay in 4 modals ([#1068](https://github.com/paypal/paypal-messaging-components/issues/1068)) ([18d4a6a](https://github.com/paypal/paypal-messaging-components/commit/18d4a6a400577f5cceb73acaf71a534c29f1cdc6))

surekhaw committed:
feat: update DE long term modal apr (#1072)

yeul committed:
refactor: update calculator padding for XO modal (#1073)

yeul committed:
chore: cut release (#1074)

semantic-release-bot committed:
chore(release): 1.63.0 [skip ci]

## [1.63.0](https://github.com/paypal/paypal-messaging-components/compare/v1.62.0...v1.63.0) (2024-04-16)

### Features

* update DE long term modal apr ([#1072](https://github.com/paypal/paypal-messaging-components/issues/1072)) ([4bfbb4d](https://github.com/paypal/paypal-messaging-components/commit/4bfbb4d5a88ecf8df6fac62060c09b70fc283c42))

### Code Refactoring

* update calculator padding for XO modal ([#1073](https://github.com/paypal/paypal-messaging-components/issues/1073)) ([9903198](https://github.com/paypal/paypal-messaging-components/commit/9903198c3da4fdd59ef68063cd640256e5a970af))

JustinDoan commented on src/library/zoid/modal/component.js:
```diff
@@ -192,10 +192,10 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
+                            event_type: 'modal_rendered',
+                            page_view_link_name: 'Calculator',
+                            page_view_link_source: 'Calculator',
+                            calculator_input: value
```
Calculator_input is expected as a string in culserv

JordanClemons replied:
`value` should already be of type string here

JordanClemons committed:
Minor fixes + remove page_loaded

jadutter committed:
refactor: font feature (#1061)

yeul committed:
refactor: allow mod XO learn more modal inside iframe (#1079)

perco12 committed:
feat: Update DE Pi30 and Ratenzahlung modals to have EN variants (#1076)

surekhaw committed:
feat: purchase protection messsage (#1075)

danzhaaspaypal committed:
chore: cut release (#1081)

semantic-release-bot committed:
chore(release): 1.64.0 [skip ci]

## [1.64.0](https://github.com/paypal/paypal-messaging-components/compare/v1.63.0...v1.64.0) (2024-04-30)

### Features

* purchase protection messsage ([#1075](https://github.com/paypal/paypal-messaging-components/issues/1075)) ([c90c280](https://github.com/paypal/paypal-messaging-components/commit/c90c28024c8743104a0497602692626a496bbd99))
* Update DE Pi30 and Ratenzahlung modals to have EN variants ([#1076](https://github.com/paypal/paypal-messaging-components/issues/1076)) ([c428184](https://github.com/paypal/paypal-messaging-components/commit/c428184c75f20b28b16aa34a0e5ab8149bde3294))

### Code Refactoring

* allow mod XO learn more modal inside iframe ([#1079](https://github.com/paypal/paypal-messaging-components/issues/1079)) ([5884791](https://github.com/paypal/paypal-messaging-components/commit/5884791a79c014604a1ac4bc9506753d912925d3))
* font feature ([#1061](https://github.com/paypal/paypal-messaging-components/issues/1061)) ([282c7fe](https://github.com/paypal/paypal-messaging-components/commit/282c7fe66b3b28fe183b6825868882fe56139707))

JordanClemons committed:
Merge conflicts

JordanClemons committed:
Fix non func snapshot tests
